data/basal.go

```diff
@@ -0,0 +1,166 @@
+package data
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	valid "github.com/tidepool-org/platform/Godeps/_workspace/src/gopkg.in/bluesuncorp/validator.v8"
+	"github.com/tidepool-org/platform/validate"
+)
+
+func init() {
+	validator.RegisterValidation(rateTag, BasalRateValidator)
```

Comment:
I'm inclined to not use const/variables for this type of thing.  Instead using just `"rate"` so it is really clear what is going on.